blocks/article-body-block/chains/article-body/default.jsx

```diff
@@ -1,22 +1,27 @@
 import React from "react";
 import PropTypes from "@arc-fusion/prop-types";
 import { useFusionContext } from "fusion:context";
-import { LazyLoad, isServerSide } from "@wpmedia/engine-theme-sdk";
+
+import { LazyLoad } from "@wpmedia/engine-theme-sdk";
 
 import {
 	Carousel,
+	Conditional,
 	formatCredits,
 	Heading,
 	HeadingSection,
 	Icon,
 	Image,
+	isServerSide,
 	Link,
 	MediaItem,
 	Paragraph,
 	usePhrases,
 	Video,
 } from "@wpmedia/arc-themes-components";
 
+import getResizeParamsFromANSImage from "./shared/get-resize-params-from-ans-image";
```

Comment:
Seems like a good candidate to move the function to Arc Themes Components? We can create a ticket to do this at a later stage if we start to see a pattern of needing to do the same logic across other blocks